tabulo.gemspec

```diff
@@ -28,8 +28,8 @@ Gem::Specification.new do |spec|
     "changelog_uri"   => "https://raw.githubusercontent.com/matt-harvey/tabulo/master/CHANGELOG.md"
   }
 
-  spec.add_runtime_dependency "tty-screen", "0.8.1"
-  spec.add_runtime_dependency "unicode-display_width", "2.2.0"
+  spec.add_runtime_dependency "tty-screen", "~> 0.8"
+  spec.add_runtime_dependency "unicode-display_width", "~> 2.2"
```

Comment:
`tty-screen` is still < v1 so even a patch bump can theoretically break things assuming semver. I'd rather keep that pinned to the latest version, which is still `0.8.1`.

As for `unicode-display_width`, happy to go with `~> 2.2` 👍🏻

Reply:
@matt-harvey sure, it makes sense! I'm gonna edit the PR now :)

Reply:
Done! Thanks for the feedback :)

Reply:
Thanks @delphaber :)